lib/node/fullnode.js

```diff
@@ -93,6 +93,7 @@ class FullNode extends Node {
       identityKey: this.identityKey,
       maxOutbound: this.config.uint('max-outbound'),
       maxInbound: this.config.uint('max-inbound'),
+      proofRPS: this.config.uint('max-proof-rps'),
```

Comment:
I think that this variable name should match the configuration name. It threw me off at first when I was trying to test this code because the rest of the variable names match the config names.

Reply:
Fixed.